macros/calibrateThrDac.py

```diff
@@ -505,18 +505,14 @@
             tgraph_scurveMeanVsThrDacForFit.SetPoint(tgraph_scurveMeanVsThrDacForFit.GetN(),thrDacVal,scurveMean)
             tgraph_scurveMeanVsThrDacForFit.SetPointError(tgraph_scurveMeanVsThrDacForFit.GetN()-1,0,scurveMeanError)
 
-        #a 4th order polynomial expanded about the lower edge of the fit range    
-        def quartic(x,par):    
-            return (par[0]*pow((x[0]-min(fitRange)),4)+par[1]*pow((x[0]-min(fitRange)),3))+par[2]*pow((x[0]-min(fitRange)),2)+par[3]*(x[0]-min(fitRange))+par[4]    
-            
```

Comment:
What was this removed?  It isn't described in the PR description.  What implications does this have on the output? E.g. the changes that you presented in the last SW dev's meeting.

Reply:
in all of the tests I have done min(fitRange) = 0, so this has no effect

it just removes a feature that could have been used

I did this because including the min(fitRange) would add complication to the storing of the function in a text file or a database

Reply:
Okay thanks for clarifying.